cardano-cli/src/Cardano/CLI/Types/Transaction.hs

```diff
@@ -0,0 +1,628 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+{- HLINT ignore "Unused LANGUAGE pragma" -}
```

Comment:
I think this pragma is not needed

Reply:
This pragma is actually needed.  Otherwise you get name conflicts.